parser/helper/helpers.go

```diff
@@ -233,6 +245,31 @@ func (h *Helper) GetActorNameFromAddress(add address.Address, height int64, key
 	}
 }
 
+// isSpecialAccountActor handles actor addresses that will fail to resolve from the node for reasons documented in each case.
+func (h *Helper) isSpecialAccountActor(add address.Address, height int64) (bool, cid.Cid, string) {
+	if ok, cid, actorName := h.isZeroAddressAccountActor(add, height); ok {
+		return true, cid, actorName
+	}
+	if ok, cid, actorName := h.isKeylessAccountActor(add, height); ok {
+		return true, cid, actorName
+	}
+	return false, cid.Undef, ""
+}
+
+// The f3yaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaby2smx7a(f067253) is a zero address actor that existed until V10.
+// Created: https://github.com/filecoin-project/lotus/blob/5750f49834deee9dfce752ff840630ae402a8b51/build/buildconstants/params_shared_vals.go#L56
+// Terminated: https://github.com/filecoin-project/lotus/blob/5750f49834deee9dfce752ff840630ae402a8b51/chain/consensus/filcns/upgrades.go#L1054
+func (h *Helper) isZeroAddressAccountActor(add address.Address, height int64) (bool, cid.Cid, string) {
+	if h.network != tools.MainnetNetwork || (add.String() != ZeroAddressAccountActorRobust && add.String() != ZeroAddressAccountActorShort) {
+		return false, cid.Undef, ""
+	}
+	version := tools.VersionFromHeight(h.network, int64(height))
+	if version.NodeVersion() < tools.V23.NodeVersion() {
+		return true, accountCid, manifest.AccountKey
+	}
```

Comment:
Is this correct here? I think it is repeated from the other function, but not required/needed